actions/app.go

```diff
@@ -73,6 +73,7 @@ func App() *buffalo.App {
 		app.GET("/about", homeHandler)
 		app.GET("/checkup", homeHandler)
 		app.GET("/collection", homeHandler)
+		app.GET("/collection/{songIdOrName}", homeHandler)
```

Comment:
Accessing from GET does not work: `could not find collection/sw/serviceWorker.js status=404`

Reply:
That error is normal, that is a result of you loading the page over HTTP